pkg/model/image_target.go

```diff
@@ -4,17 +4,14 @@ import (
 	"fmt"
 	"sort"
 
-	"github.com/docker/distribution/reference"
-
 	"github.com/windmilleng/tilt/internal/container"
 	"github.com/windmilleng/tilt/internal/sliceutils"
 )
 
 type ImageTarget struct {
-	ConfigurationRef container.RefSelector
-	DeploymentRef    reference.Named
-	BuildDetails     BuildDetails
-	MatchInEnvVars   bool
+	Refs           container.RefSet
```

Comment:
out of curiosity, why group them together into a sub-struct? it doesn't seem like you ever use them together as a group

Reply:
it might make sense to add ConfigurationRef() etc accessors

Reply:
in a future PR i pass refsets around to builders -- i don't feel strongly about it, it just seemed a way to reduce the number of args to keep track of 🤷‍♀